addons/knobs/configure.js

```diff
@@ -1,11 +1 @@
-import {withKnobs as addonWithKnobs} from '@storybook/addon-knobs';
-
-function withKnobs(opts) {
-	return addonWithKnobs({
-		timestamps: true, // Doesn't emit events while user is typing.
```

Comment:
Don't we need this option anymore?

Reply:
This code is moved to storybook configuration files (@enact/sampler/.storybook/preview.js)